config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal.yaml

```diff
@@ -0,0 +1,133 @@
+     bmhLabelSelector:
+       app: openstack
+     ctlplaneInterface: enp1s0
+     ctlplaneGateway: 192.168.111.1
+     ctlplaneNetmask: 255.255.255.0
+     domainName: osptest.openstack.org
+     bootstrapDns:
+       - 192.168.111.1
+     dnsSearchDomains:
+       - osptest.openstack.org
+     baremetalHosts: 2
+  nodeTemplate:
+    env:
+      - name: ANSIBLE_FORCE_COLOR
+        value: "True"
+      - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+        value: "True"
+    nodes:
+       edpm-compute-0:
+         nova: {}
+         hostName: edpm-compute-0
+         ansibleHost: 192.168.111.100
+         node:
+           ansibleVars:
+             ctlplane_ip: 192.168.111.100
+             internal_api_ip: 172.17.0.100
+             storage_ip: 172.18.0.100
+             tenant_ip: 172.19.0.100
+             fqdn_internal_api: edpm-compute-0.example.com
+       edpm-compute-1:
+         hostName: edpm-compute-1
+         ansibleHost: 192.168.111.101
+         nova: {}
+         node:
+           ansibleVars:
+             ctlplane_ip: 192.168.111.101
+             internal_api_ip: 172.17.0.101
+             storage_ip: 172.18.0.101
+             tenant_ip: 172.19.0.101
+             fqdn_internal_api: edpm-compute-1.example.com
+    managementNetwork: ctlplane
+    ansibleUser: cloud-admin
+    ansiblePort: 22
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+    ansibleVars:
+       service_net_map:
+         nova_api_network: internal_api
+         nova_libvirt_network: internal_api
+       edpm_chrony_ntp_servers:
+         - clock.redhat.com
+       # edpm_network_config
+       # Default nic config template for a EDPM compute node
+       # These vars are edpm_network_config role vars
+       edpm_network_config_template: templates/single_nic_vlans/single_nic_vlans.j2
+       edpm_network_config_hide_sensitive_logs: false
+
+       # These vars are for the network config templates themselves and are
+       # considered EDPM network defaults.
+       neutron_physical_bridge_name: br-ex
+       neutron_public_interface_name: eth0
+       ctlplane_mtu: 1500
+       ctlplane_subnet_cidr: 24
+       ctlplane_gateway_ip: 192.168.111.1
+       ctlplane_host_routes:
+       - ip_netmask: 0.0.0.0/0
+         next_hop: 192.168.111.1
+       external_mtu: 1500
+       external_vlan_id: 44
+       external_cidr: '24'
+       external_host_routes: []
+       internal_api_mtu: 1500
+       internal_api_vlan_id: 20
+       internal_api_cidr: '24'
+       internal_api_host_routes: []
+       storage_mtu: 1500
+       storage_vlan_id: 21
+       storage_cidr: '24'
+       storage_host_routes: []
+       tenant_mtu: 1500
+       tenant_vlan_id: 22
+       tenant_cidr: '24'
+       tenant_host_routes: []
+       role_networks:
+       - InternalApi
+       - Storage
+       - Tenant
+       networks_lower:
+         External: external
+         InternalApi: internal_api
+         Storage: storage
+         Tenant: tenant
```

Comment:
we shoudl be able to generate this form the hostnetworks or networkattachmetns. and not need to specify it in the sample